src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,8 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < 3; i++) {
+            System.out.println(new Lottery().getRandomBall().toString());
+        }
```

Comment:
1. Please extract creation of lottery object from cycle, we may obtain required result creating only 1 object but not new for each cycle iteration
2.  System.out.println method automatically calls .toString() method of object passed, no need to explicitly indicate it
```suggestion
        Lottery lottery = new Lottery();
        for (int i = 0; i < 3; i++) {
            System.out.println(lottery.getRandomBall());
        }
```